README.md

```diff
@@ -1 +1,44 @@
-#CINEMA
+#Football-tickets
```

Comment:
It's up to you, but I would advise using a different template for the readme. Not it looks similar to other projects visually.

Reply:
okey